nextflow.config

```diff
@@ -79,9 +75,9 @@ manifest {
   name = 'nf-core/mag'
   description = 'Assembly, binning and annotation of metagenomes'
   homePage = 'https://github.com/nf-core/mag'
-  pipelineVersion = '1.0dev'
+  version = '1.0.0'
   mainScript = 'main.nf'
-  nextflowVersion = '>=0.30.0'
+  nextflowVersion = '>=0.32.0'
 }
 
```

Comment:
You might want to use the novel nf-core/configs here to be compatible with other pipelines. This will come in via the pipeline syncing feature soonish, but doesn't harm to have it in for your first release already :-) 

https://github.com/nf-core/tools/blob/dev/nf_core/pipeline-template/%7B%7Bcookiecutter.name_noslash%7D%7D/nextflow.config#L40-L63 

https://github.com/nf-core/tools/blob/d0bc828764d7214d48964327ef431a54156b87a1/nf_core/pipeline-template/%7B%7Bcookiecutter.name_noslash%7D%7D/nextflow.config#L36